src/tests/unit/tests/DetailsView/components/__snapshots__/assessment-test-view.test.tsx.snap

```diff
@@ -3,13 +3,13 @@
 exports[`AssessmentTestView assessment view, isScanning is false 1`] = `
 "<Fragment>
   <ScanIncompleteWarning deps={{...}} warnings={[undefined]} warningConfiguration={{...}} test={-1} />
-  <AssessmentView deps={{...}} isScanning={false} isEnabled={false} assessmentNavState={{...}} assessmentInstanceTableHandler={{...}} assessmentData={{...}} currentTarget={{...}} prevTarget={[undefined]} assessmentDefaultMessageGenerator={[undefined]} assessmentTestResult={{...}} featureFlagStoreData={{...}} pathSnippetStoreData={{...}} switcherNavConfiguration={{...}} />
+  <FlaggedComponent enableJSXElement={{...}} disableJSXElement={{...}} featureFlag=\\"reflowUI\\" featureFlagStoreData={{...}} />
```

Comment:
this looks odd. are we using `debug` here instead of `getElement`?

Reply:
Ideally we should see the different components being passed into enable and disable element.

Reply:
Checked and the original test was not using `getElement` but was using `debug`. I can put out a new PR with the change. It shows the different components with `getElement`